### ISSUE TITLE
Error building with ghc-9

### ISSUE THREAD

**Issue by locallycompact** [luke-clifton/shh#51]
Hi, I'm running into this. I'm not sure what is expected here or what to change.

```
shh                              >
shh                              > /run/user/1000/stack-d53fff64325d5f36/shh-0.7.1.0/src/Shh/Internal.hs:684:18: error:
shh                              >     • Couldn't match type: [ByteString]
shh                              >                      with: forall a. (HasCallStack, Command a) => a
shh                              >       Expected: Cmd -> [ByteString]
shh                              >         Actual: [ByteString] -> [ByteString]
shh                              >     • In the expression: toArgs
shh                              >       In an equation for ‘displayCommand’: displayCommand = toArgs
shh                              >     |
shh                              > 684 | displayCommand = toArgs
shh                              >     |                  ^^^^^^
```

**Comment by luke-clifton** [comment]
Thanks for the report. This seems to be due to the new [simplified subsumption](https://gitlab.haskell.org/ghc/ghc/-/wikis/migration/9.0#simplified-subsumption) in GHC 9. The fix is easy enough. There are a few other breakages in GHC 9 which I need to fix as well. Will probably have a patch in a few hours.

**Comment by luke-clifton** [comment]
Fixed in #52

**Comment by luke-clifton** [comment]
Released 0.7.1.1 on Hackage which works with GHC 9
